### DISCUSSION TITLE
Add a confirmation dialog that pops up when a user trying to merge a PR that is out of date

### DISCUSSION THREAD

**Discussion by a-asharapova** [GitbookIO/community#810] [Feature Requests]
As a user, I want to see a confirmation dialog when I press the Merge **button** on an out-of-date PR.

**Comment by emmerich** [comment]
That's a cool idea. We're planning on building more controls around change requests (e.g. being able to say "every CR requires one approval"). Would having a "require all change requests to be up-to-date" be a good solution to this? That way it fits well in our approach for CR checks.

**Reply by a-asharapova** [reply]
@emmerich It would, but ideally admins should be able to disable and enable this check.
